### PR TITLE
Allow us to log in to the test IDCS account in staging.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
Log into the test IDCS account in staging - while maintaining code compatibility for dev.

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by hmhsu — APPROVED** [review]

File: universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java [hunk 1]
```diff
@@ -259,6 +259,7 @@ private void truncateTables() {
         Account.class,
         Applicant.class,
         Application.class,
+        models.Version.class,
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java:262]: Looks like you need at least one active version.

Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java:262]: Ah you're right - this didn't impact the tests I was running on staging.  🤦

**Comment by ndmckinley** [comment]
Okay - let's see if this works...

**Comment by ndmckinley** [comment]
Okay... gonna merge this.  Still very suspicious of the /dev/seed issue...

**Pull request merged**